src/models/reporting.ts

```diff
@@ -14,7 +14,8 @@
  *  limitations under the License.
  *
  */
-
+// eslint-disable-next-line import/named
+import { Task, TaskMeta } from 'vitest';
```

Comment:
Does it apply here as well?
> `import * as vitest from 'vitest';`

Reply:
I tried to apply this change but faced with lint error when `interface RPTaskMeta extends vitest.TaskMeta`
Error `'TaskMeta' not found in imported namespace 'vitest' import/namespace`